src/Elastic.Apm.Extensions.Logging/ApmErrorLogger.cs

```diff
@@ -42,7 +42,7 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
 
 			if (_agent is ApmAgent apmAgent && exception != null)
 			{
-				errorLog.StackTrace = StacktraceHelper.GenerateApmStackTrace(exception, null, "CaptureErrorLogsAsApmError",
+				errorLog.StackTrace = StacktraceHelper.GenerateApmStackTrace(exception, _agent.Logger, "CaptureErrorLogsAsApmError",
```

Comment:
minor nitpick: 

```suggestion
				errorLog.StackTrace = StacktraceHelper.GenerateApmStackTrace(exception, apmAgent.Logger, "CaptureErrorLogsAsApmError",
```

It makes no difference either way, but reads more intuitively to me, since the other passed values come from `apmAgent`